data/base-town.yaml

```diff
@@ -630,6 +630,10 @@ Steelclaw Clan:
     id: 19399
   trader_outpost:
     id: 2874
+  metal_repair:
+    id: 9639
+    name: Granzer
+  deposit: *shard_deposit
```

Comment:
To confirm, this doesn't need to be?
```yaml
deposit:
  << : *shard_deposit
```

Reply:
Mahtra clarified this syntax also works in Discord, they were having issues posting reply to this comment.